xml/hwbp_registers.xml

```diff
@@ -1252,11 +1252,13 @@ same project unless stated otherwise.
 
         This register is accessible as {csr-tdata1} when {tdata1-type} is 5.
 
-        This trigger may fire on up to XLEN of the Exception Codes defined in
-        `mcause` (described in the Privileged Spec, with Interrupt=0). Those
-        causes are configured by writing the corresponding bit in {csr-tdata2}.
-        (E.g.  to trap on an illegal instruction, the debugger sets bit 2 in
-        {csr-tdata2}.)
+        This trigger can match when a trap is taken due to an exception.
```

Comment:
This feels like a very wish-washy statement.

How about:
This trigger matches when an exception is taken and the bit corresponding to the exception is set in tdata2.

Probably requires rewriting the paragraph below for it all to make sense.

Reply:
> This feels like a very wish-washy statement.

I agree.  I didn't like the phrasing but I was trying to match the existing itrigger description which was [updated last year](https://github.com/riscv/riscv-debug-spec/commit/92835a85352d2b5e087ca953c164181367078fe8).  I was trying to limit the scope of this change but maybe we should update itrigger, too.